File: 백준/Bronze/24083. 短針 （Hour Hand）/短針 （Hour Hand）.cc
#include <iostream>

using namespace std;
int main()
{
    int a, b;
    cin >> a;
    cin >> b;

    int result = (a + b) % 12;
    if (result == 0)
    {
        cout << 12 << endl;
    }
    else
    {
        cout << result << endl;
    }
}

File: 백준/Bronze/24083. 短針 （Hour Hand）/短針 （Hour Hand）.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());

    let mut input = reader.lines();

    if let Some(Ok(line)) = input.next() {
        let a = line.parse::<usize>().unwrap();
        if let Some(Ok(line)) = input.next() {
            let b = line.parse::<usize>().unwrap();
            let result = (a + b) % 12;
             if result ==0{
                writeln!(writer,"{}",12).unwrap();
             }else{
                writeln!(writer,"{}",result).unwrap();

             }
        }
    }
}
